src/lib/mf.rs

```diff
@@ -38,7 +38,7 @@ use std::hash::{Hash, Hasher};
 
```

Comment:
Since I can't comment directly on a commit message:
This does ~~leave~~ lead to a speed-up...

Reply:
~~put~~ but 25% seems not uncommon

Reply:
OK, I can only fix this when I'm rebasing.